css/_components.scss

```diff
@@ -51,3 +51,69 @@
         }
     }
 }
+
+
+
+/**
+ * The background-video mixin styles an edge-to-edge background video. Because 
+ * elements within .hero are absolutely positioned, .hero’s height gets wiped 
+ * out. We assign it a faux-height via a ::before pseudo element.
+ *
+ * Emmet: .background-video>.hero>.panel{$1}+video
+ */
+@mixin background-video() {
+    .container {
+        margin-left: auto;
+        margin-right: auto;
+        max-width: 1140px;
+        padding-left: 20px;
+        padding-right: 20px;
+        position: relative;
+        z-index: 2;
+    }
+
+    .hero {
+        overflow: hidden;
```

Comment:
The component isn't working properly for me; that is, the height of the video is being solely controlled by the text/padding, and not the pseudo element.

I believe that this line is the issue; setting overflow: hidden here is chopping things down. (duplicated in Firefox and Chrome.)